### ISSUE TITLE
Package build instructions insufficient if package cloned directly

### ISSUE THREAD

**Issue by markusbaumeister** [gap-packages/profiling#41]
The README contains the lines:

The package should be built using the commands:
./configure
make

This should be replaced by (if one clones it directly into the package repository):

The package should be built using the commands:
./autogen.sh
./configure
make

**Comment by fingolfin** [comment]
I'd argue that people who "clone directly" should know what they are doing...

**Comment by markusbaumeister** [comment]
Is the README actually useful for other groups of people?

**Comment by fingolfin** [comment]
Yes, for people who get a released version of profiling (either as part of a full GAP release, or as a separate download).
